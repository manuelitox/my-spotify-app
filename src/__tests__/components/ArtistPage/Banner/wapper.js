import React from 'react'
import { shallow } from 'enzyme'

import BannerWrapper from 'components/ArtistPage/Banner/wrapper'

describe('Component | BannerWrapper', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <BannerWrapper />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})