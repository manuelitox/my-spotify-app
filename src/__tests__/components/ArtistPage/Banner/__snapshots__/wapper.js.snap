// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | BannerWrapper should render a component 1`] = `
<styled.div>
  <BannerWrapper
    genres={
      Array [
        "hip pop",
        "neo soul",
        "pop",
        "r&b",
        "urban contemporary",
      ]
    }
    name="Alicia Keys"
    photo="https://i.scdn.co/image/05e25d030a1caa1611b1e7248d89b557d9867258"
  />
</styled.div>
`;
